auslib/test/test_db.py

```diff
@@ -1003,7 +1003,13 @@ def testWhitelistIncluded(self):
         rel_name = 'ab'
         rule_id = 6
 
-        releases = self.releases.getReleaseInfo()
+        with mock.patch('auslib.db.dbo') as mock_dbo:
+            mock_dbo.releases.t = self.db.releases.t
+            mock_dbo.rules.t = self.db.rules.t
+            mock_dbo.releases.name = self.releases.name
+            mock_dbo.rules.mapping = self.rules.mapping
+            mock_dbo.rules.whitelist = self.rules.whitelist
+            releases = self.releases.getReleaseInfo()
```

Comment:
I think you may need to replace a few more things here. If you don't want to fuss with this any further, you can just remove these tests though. As I said on IRC, the real bug here is that db.py is using the dbo at *all*, which is not your fault.